### PR TITLE
don't get gravatar for users without emails

### PR DESCRIPTION
**Pull request by sebastianovide**

### CODE REVIEW

**Comment by sebastianovide** [comment]
for #320

File: src/utils/gravatar.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import User from "types/User";
+
+declare global {
+    interface Window {
+        userFromGravatar:any;
+    }
+}
+
+export function addGravatarInfo(currentUser: User) {
+  if (_.isEmpty(currentUser.email)) return;
```
Comment (alastairdglennie) [inline src/utils/gravatar.ts:12]: Should we error this? Seems pretty fatal for the normal flow?

File: src/utils/__tests__/gravatar.test.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "",
+            "",
+            "",
+            null,
+            "",
+            []
+        );
+    });
+
+    it("gravatar are ingnored for users without emails", () => {
```
Comment (alastairdglennie) [inline src/utils/__tests__/gravatar.test.ts:25]:
```suggestion
    it("gravatar are ignored for users without emails", () => {
```

**Review by alastairdglennie — APPROVED** [review]

File: src/utils/gravatar.ts (hunk 1 above)
Comment (sebastianovide) [reply src/utils/gravatar.ts:12]: well... you can still login without sharing the email... people are getting more conscious about privacy and less happy about sharing emails...

**Pull request merged**